assets/js/helpers/storage.js

```diff
@@ -22,3 +22,33 @@ export function storageAvailable(type) {
         );
     }
 }
+
+// Based on https://gist.github.com/anhang/1096149
+export function setWithExpiry({ type, key, data, expiryInMinutes }) {
+    if (!storageAvailable) {
+        return;
+    }
+    const expirationMs = expiryInMinutes * 60 * 1000;
+    const record = {
+        value: JSON.stringify(data),
+        timestamp: new Date().getTime() + expirationMs
+    };
+    window[type].setItem(key, JSON.stringify(record));
+    return data;
+}
+
+export function getWithExpiry({ type, key }) {
+    if (!storageAvailable || !window[type].getItem(key)) {
```

Comment:
Should `storageAvailable` be a function call?

Reply:
Oops, yes!